bin/lock-wikis

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script can be used to set the read-only setting for a list of wikis.
+# 
+# `DOMAIN_FILE_LIST` is just the path to a plain-text file containing a list of wikibase.cloud domains, each line containing 1 wiki domain
```

Comment:
Suggestion: Instead of forcing users to provide a file, we could also use pipes, allowing users to use any mechanism they want, e.g.:

```
cat domains.txt | lock-wikis
echo "testthis.wikibase.cloud" | lock-wikis
kubectl exec "whatever" | lock-wikis
./some-command-from-the-future | lock-wikis
```

Reply:
yes I even started writing it like that but I just used $* to accept the list - and IIRC theres *some* limit to number of arguments or something? I just went for boring but safe stuff here (according to my limited bash scripting skills)

Reply:
> yes I even started writing it like that but I just used $* to accept the list 

I think `read` should give us piping in unlimited ways:

```sh
while read line; do
  echo "i'm a parrot: $line`
done
```